keyboards/meetlab/kalice/info.json

```
@@ -0,0 +1,128 @@
+{
+    "manufacturer": "lucky_studio",
+    "keyboard_name": "Kalice",
+    "maintainer": "yuezp",
+    "bootloader": "stm32duino",
```

You can replace the bootloader and process with just: 
```suggestion
    "development_board": "bluepill",
```

---

thanks